content/en/guides/os-commercial-ecosystem.md

```diff
@@ -0,0 +1,180 @@
+ 
+## Instill confidence for commercial adoption
+
+As an open source software project grows, [a study shows](http://www.ifosslr.org/ifosslr/article/view/64) it reaches an inflection point at which corporations want to participate, but aren’t necessarily comfortable with the intellectual property regime (or lack thereof) in the open source project. These practices help instill confidence in companies that may be hanging out on the fringes, waiting to commit to the project.
+
+* IP and trademark management. Open source licenses and policies, along with trademarks, can have a huge impact on whether a project achieves commercial adoption. There are many [free resources](https://www.linux.com/news/free-tools-driving-open-source-project-success) available for managing these.
+
+* Independent governance. For projects at a certain stage of growth, it becomes critically important to establish independent governance and neutral project assets. Remember that it is critical to involve people who are qualified to supply business and legal governance of projects as well as people who can supply technical governance. For example, a person with technical skills may have an understanding of Inbound Contribution Guidelines, while a person with business credentials might be qualified to set policies regarding trademarks.
+
+
+*“In fostering a healthy commercial ecosystem, have a clear vision of how your project is governed, specifying what is and isn't a part of the particular project. Have a clear set of interfaces functioning as places where your commercial ecosystem can plug in, or attach to the project. As you start developing an ecosystem, you want companies to feel safe in the work that they are doing with you.”* – Sarah Novotny, FOSS strategy Azure 
+
+* Supported, secure, and reliable code and infrastructure. How much benefit has Red Hat derived from offering hardened support for open source software? A huge amount. In addition to auditing your code for reliability and security, focus on offering support and leverage community support options such as forums. Many firms, such as Black Duck, also offer open source security and reliability audits.
+
+* PR and marketing support. Does your project have a blog and is there a plan surrounding it for marketing and public relations? Creating your own content around your project and complementing that effort with outreach to the media along with marketing can get your project talked about, and adopted. Many commercial open source projects experience conversion ratios (as measured by the percentage of downloaders who buy something) that are not high when compared to proprietary software products, so low-cost and very scalable marketing functions are key to profitability.
```

Comment:
Might be a given to most of us, but probably still worth adding explicitly: talk about your project at open source events

```suggestion
* PR and marketing support. Does your project have a blog and is there a plan surrounding it for marketing and public relations? Creating your own content around your project and complementing that effort with presentations at open source events as well as outreach to the media along with marketing can get your project talked about, and adopted. Many commercial open source projects experience conversion ratios (as measured by the percentage of downloaders who buy something) that are not high when compared to proprietary software products, so low-cost and very scalable marketing functions are key to profitability.
```

Reply:
right! thanks! :)